README.md

```diff
@@ -99,8 +99,8 @@ Parameters: none
 
 Platform requirements:
 
-* [PHP] 8.0 or later (tested up to 8.1)
-* [MediaWiki] 1.35 or later (tested up to 1.39 and master)
+* [PHP] 8.0 or later (tested up to 8.3)
+* [MediaWiki] 1.35 or later (tested up to 1.43 and master)
```

Comment:
If we're dropping CI for 1.35, then we should remove support for it completely. Otherwise, we have no way to verify that 1.35 works with any new changes.